src/test/java/com/rednavis/vaadin/showcase/backend/BackendServiceTest.java

```diff
@@ -0,0 +1,24 @@
+package com.rednavis.vaadin.showcase.backend;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+
+@TestInstance(Lifecycle.PER_CLASS)
+public class BackendServiceTest {
+  
+  private BackendService service;
+  
+  @BeforeAll
+  public void init() {
+    service = new BackendService();
+  }
+
+  @Test
+  public void should_test_get_employees() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The JUnit 4 test method name 'should_test_get_employees' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4808499)